pkgs/development/python-modules/meshcat/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ipython
+, u-msgpack-python
+, numpy
+, tornado
+, pyzmq
+, pyngrok
+, pillow
+}:
+
+buildPythonPackage rec {
+  pname = "meshcat";
+  version = "0.3.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-LP4XzeT+hdByo94Bip2r9WJvgMJV//LOY7JqSNJIStk=";
+  };
+
+  postPatch = ''
+    sed -i '/PYTHONPATH/d' src/meshcat/servers/zmqserver.py
+  '';
+
+  propagatedBuildInputs = [
+    ipython
+    u-msgpack-python
+    numpy
+    tornado
+    pyzmq
+    pyngrok
+    pillow
+  ];
+
+  # requires a running server
```

Comment:
What kind of server?

Reply:
In fact, a [MeshCat viewer](https://github.com/rdeits/meshcat/#meshcat).